File: inflearn/javascript/1-02/main.js
const ps = require('../modules/ps');

const solution = (input1, input2) => {
  let answer = 0;
  input1 = input1.toLowerCase(); // 문자열을 소문자로 변환
  for (let i = 0; i < input1.length; i++) {
    if (input1[i] === input2) {
      answer++;
    }
  }
  return answer;
};

ps.main(async () => {
  let input1 = await ps.readLine();
  let input2 = await ps.readLine();

  console.log(solution(input1, input2));
});

File: inflearn/java/1-02/Main.java
import java.util.*;

public class Main {
    public static String solution(String str) {
        String answer = "";
        for (int i = 0; i < str.length(); i++) {
            char t = str.charAt(i);
            if (t >= 'a')  // 아스키 코드로 비교
                answer += Character.toUpperCase(t);
            else
                answer += Character.toLowerCase(t);
        }
        return answer;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input = sc.next();

        System.out.println(solution(input));
        sc.close();
    }
}
